python/ray/serve/_private/router.py

```diff
@@ -331,6 +332,35 @@ def __init__(
         self._pending_requests_to_fulfill: Deque[PendingRequest] = deque()
         self._pending_requests_to_schedule: Deque[PendingRequest] = deque()
 
+        # Prepare scheduler metrics.
+        self._actor_name: str = self._get_actor_name_GCS_UNSAFE()
+
+        self.num_scheduling_tasks_gauge = metrics.Gauge(
+            "serve_num_power_of_two_choices_scheduling_tasks",
+            description=(
+                "The number of power-of-two-choices scheduling tasks "
+                "run by the router."
```

Comment:
Sorry, just saw, but I'd suggest the dropping `power_of_two_choices` here from the name (and the other metric) as it is exposing implementation details.

Reply:
I dropped `power_of_two_choices`.

Reply:
The number of scheduling tasks is also an implementation detail. I added these metrics to the documentation, but we may want to remove it. I'm not sure how informative they are to end users.